src/CosmosDB/CosmosDB/help/Get-AzCosmosDbClientEncryptionKey.md

```diff
@@ -0,0 +1,150 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.CosmosDB.dll-Help.xml
+Module Name: Az.CosmosDB
+online version:
+schema: 2.0.0
+---
+
+# Get-AzCosmosDbClientEncryptionKey
+
+## SYNOPSIS
+Gets the CosmosDB Client Encryption Key.
+
+## SYNTAX
+
+### ByNameParameterSet (Default)
+```
+Get-AzCosmosDbClientEncryptionKey -ResourceGroupName <String> [-Name <String>]
+ [-DefaultProfile <IAzureContextContainer>] -AccountName <String> -DatabaseName <String> [<CommonParameters>]
+```
+
+### ByParentObjectParameterSet
+```
+Get-AzCosmosDbClientEncryptionKey [-Name <String>] -ParentObject <PSSqlDatabaseGetResults>
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Get-AzCosmosDbClientEncryptionKey** cmdlet gets the list of all existing CosmosDB Client Encryption Key for a given ResourceGroupName, AccountName and DatabaseName and gets a single CosmosDB Client Encryption Key for a given ResourceGroupName, AccountName, DatabaseName and ClientEncryptionKeyName.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Get-AzCosmosDbClientEncryptionKey -AccountName {accountName} -ResourceGroupName {resourceGroupName} -DatabaseName {databaseName}
+
+Name                     : {clientEncryptionKeyName}
+Id                       : Id
+Resource                 : Microsoft.Azure.Commands.CosmosDB.Models.PSSqlClientEncryptionKeyGetPropertiesResource
+```
+
```

Comment:
Please add description here. Please follow this pattern: https://github.com/Azure/azure-powershell/blob/98e720fbf67ccdd2f84e6a3e2a36d930708e5bce/src/KeyVault/KeyVault/help/Add-AzKeyVaultKey.md?plain=1#L131-L153

Reply:
Fixed.